spark/src/test/java/org/apache/iceberg/spark/source/TestORCWrite.java

```diff
@@ -0,0 +1,99 @@
+package org.apache.iceberg.spark.source;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Lists;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.iceberg.*;
```

Comment:
The Iceberg code style doesn't allow using the wildcard import. You might want to change your IDE setting to disable the import optimization.